src/Units/Unit.java

```diff
@@ -211,7 +211,7 @@ public double alterHP(int change)
     double before = HP;
     // Change the unit's health, but don't grant more
     // than 10 HP, and don't drop HP to zero.
-    HP = Math.max(1, Math.min(10, getHP() + change));
```

Comment:
Map damage is now up to 9% more effective?

Reply:
Matches the carts.